packages/terra-show-hide/src/clinical-lowlight-theme/ShowHideButton.module.scss

```diff
@@ -0,0 +1,30 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .clinical-lowlight-theme {
+    --terra-show-hide-v1-button-color: #007ee0;
+    --terra-show-hide-v1-button-font-size: 1rem;
+    --terra-show-hide-v1-button-line-height: 1.429;
+    --terra-show-hide-v1-button-padding-bottom: 0.21428rem;
+    --terra-show-hide-v1-button-padding-left: 0;
+    --terra-show-hide-v1-button-padding-right: 0;
+    --terra-show-hide-v1-button-padding-top: 0.21428rem;
+    --terra-show-hide-v1-button-text-decoration: none;
+    --terra-show-hide-v1-button-icon-height: 0.7143em;
+    --terra-show-hide-v1-button-icon-width: 0.7143em;
+    --terra-show-hide-v1-button-icon-margin-left: 0.28571em;
+    --terra-show-hide-v1-button-focus-background-color: transparent;
+    --terra-show-hide-v1-button-focus-border-radius: 0.2142rem;
+    --terra-show-hide-v1-button-focus-box-shadow: 0 0 1px 3px #004c76, 0 0 7px 4px #004c76;
+    --terra-show-hide-v1-button-focus-color: #007ee0;
+    --terra-show-hide-v1-button-focus-text-decoration: underline;
+    --terra-show-hide-v1-button-hover-background-color: none;
+    --terra-show-hide-v1-button-hover-color: #003f61;
+    --terra-show-hide-v1-button-hover-text-decoration: underline;
+    --terra-show-hide-v1-button-active-color: #003f61;
```

Comment:
These versioning identifiers are probably worth re-evaluating down the road. I wouldn't make any changes on this PR, but wanted to point it out.

Reply:
can we log an issue for this?